tests/gpu_provers.rs

```diff
@@ -94,13 +94,15 @@ pub fn test_parallel_prover() {
     // Have higher prio proof wait long enough to interrupt lower
     thread::sleep(Duration::from_millis(2000));
 
-    {
+    let higher_thread = thread::spawn(move || {
```

Comment:
is this different from leaving the higher prio in the originating thread and moving the lower to its own thread?

Reply:
No its not, was just playing with it